### PR TITLE
made changes to match the automarker

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)
1. **Asking the learner to create an object for morsecode.** 
The reason is the automaker iterates over an object and I bumped into a learner's code where the morsecode dictionary was created inside a helper function and the returned value was already filtered. e.g something like this:
```
function decoder(key, value) {
  const morseDecoder = {
    ".-": "A",
    "-...": "B",
    ....
   if (value) {
       return Object.keys(morseDecoder).find((key) => morseDecoder[key] === value);
   } else {
      return morseDecoder[key];
  ...
```
2. **I changed the assertion messages to match those on the automaker.** 

_I am unsure how this affects the other languages, please let me know if we'd need to specify these instructions for the js language only._

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changes in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/morse-code/_index.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ Your directory structure should look like this:
 
 ## Instructions
 
-Create two functions:
+- Create an object of that contains morsecode. See resources below for examples. Make sure that you create only one object.
```
Comment (Ngoakor12) [inline content/projects/morse-code/_index.md:62]: The first sentence on the first point doesn't seem right. Shouldn't it be `Create an object that contains morsecode characters`?

File: content/projects/morse-code/_index.md [hunk 2]
```diff
@@ -80,12 +81,10 @@ morseCodeToLetters(".... .. / - .... . .-. .");
 
 Make sure to cater for punctuation marks and all special characters.
 
-Include the following assertions in your code:
+- Include the following assertions in your code:
```
Comment (Ngoakor12) [inline content/projects/morse-code/_index.md:84]: I'm not sure why this is a list item with nested list items

File: content/projects/morse-code/_index.md [hunk 3]
```diff
@@ -80,12 +81,10 @@ morseCodeToLetters(".... .. / - .... . .-. .");
...
 
-- Assert that the output and input both have the same number of characters represented. Use the string `the number of input and output characters is not the same` when both input and output characters are not equal.
-- Assert that there are the right number of spaces represented in the output. Use the string `the number of input and output space characters is not the same` when both input and output space characters are not equal.
-
-Make sure that when you do this, your code remains clean and dry.
+  - Assert that the output and input both have the same number of characters represented. Use the string `the number of letter characters is not equal to the number of morse code characters` when both input and output characters are not equal.
+  - Assert that there are the right number of spaces represented in the output. Use the string `the number of letter space characters is not equal to the number of morse code space characters` when both input and output space characters are not equal.
```
Comment (Ngoakor12) [inline content/projects/morse-code/_index.md:87]: The current automarker for this project is deactivated with the assertions strings being one of the problems? Any idea on how to make them less weird/confusing.

Check out the related config on the automarker projects repo to see why it was deactivated.

It's fine if you can't come up with anything now.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

File: content/projects/morse-code/_index.md (hunk 1 above)
Comment (sheenarbw) [inline content/projects/morse-code/_index.md:62]: The English isn't good.

Also this seems spoonfeedy. Can't we rather let the learner think for themselves?

File: content/projects/morse-code/_index.md (hunk 3 above)
Comment (sheenarbw) [reply content/projects/morse-code/_index.md:87]: Just tell them to think about what error messages would be useful and then they can write those. Coming up with clear error messages is something a developer needs to be able to do.

Spoonfeeding people does make the code easier to review. But  spoonfeeding is bad for education.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
I think this PR might be a big misguided. What problem are we trying to solve here?

File: content/projects/morse-code/_index.md (hunk 3 above)
Comment (FaithMo) [reply content/projects/morse-code/_index.md:87]: > The current automarker for this project is deactivated with the assertions strings being one of the problems? Any idea on how to make them less weird/confusing.
> 
> Check out the related config on the automarker projects repo to see why it was deactivated.
> 
> It's fine if you can't come up with anything now.

Oh I see, cool no problem. I didn't have the whole picture. Let me close it for now!

**Pull request closed without merging**